config/config.go

```diff
@@ -335,6 +326,14 @@ func (t *TCF2) PurposeEnforced(purpose consentconstants.Purpose) (value bool) {
 	return t.PurposeConfigs[purpose].EnforcePurpose
 }
 
+// PurposeEnforcementAlgo
```

Comment:
IMHO it's better to omit a comment then include a blank one.

Reply:
Oops I meant to add a description here